src/settings.py

```diff
@@ -206,12 +206,16 @@ class GlobalSettings:
 
     # interactive learning parameters
     interactive_num_ensemble_members = 10
-    interactive_action_strategy = "glib"
-    interactive_query_policy = "strict_best_seen"
-    interactive_score_function = "frequency"
-    interactive_score_threshold = 0.5
+    interactive_action_strategy = "greedy_lookahead"
```

Comment:
is there a reason you want to change these defaults? (doesn't matter much, just wondering)

Reply:
the combination of frequency + strict_best_seen is just not very good